x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/overrides_validation.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+const INDETERMINATE_FIELD_PLACEHOLDER = '?';
+const FRACTIONAL_SECOND_SEPARATORS = ':.,';
+
+const VALID_LETTER_GROUPS = {
+  'yyyy': true,
+  'yy': true,
+  'M': true,
+  'MM': true,
+  // The simple regex here is based on the fact that the %{MONTH} Grok pattern only matches English and German month names
+  'MMM': true,
+  'MMMM': true,
+  'd': true,
+  'dd': true,
+  // The simple regex here is based on the fact that the %{DAY} Grok pattern only matches English and German day names
```

Comment:
This comment relates to the values in the ES Java code which are regular expressions, but since your values are just `true` the comment is unnecessary.  The one on line 17 can also go.

Reply:
Ah, thanks for catching those - I'll remove them 👍 😄

Reply:
Removed in 2adb1002d6ef80fb467803db00d1273545636951